object.c

```diff
@@ -1251,14 +1255,44 @@ rb_obj_frozen_p(VALUE obj)
 /*
  * Document-class: NilClass
  *
- *  The class of the singleton object <code>nil</code>.
+ * The class of the singleton object +nil+.
+ *
+ * Several of its methods act as operators:
+ *
+ * - #&
+ * - #|
+ * - #===
+ * - #=~
+ * - #^
+ *
+ * Others act as converters, carrying the concept of _nullity_
+ * to other classes:
+ *
+ * - #rationalize
+ * - #to_a
+ * - #to_c
+ * - #to_h
+ * - #to_r
+ * - #to_s
+ *
+ * Another method provides inspection:
+ *
+ * - #inspect
+ *
+ * Finally, there is this query method:
+ *
+ * - #nil?
+ *
  */
 
 /*
  *  call-seq:
- *     nil.to_s    -> ""
+ *    to_s -> ''
+ *
+ *  Returns an empty String:
+ *
+ *    nil.to_s # => ""
  *
```

Comment:
The indentation of this section is 2 spaces, while it's 1 for everything else, I'm not sure why there's an inconsistency here

Reply:
Fixed.